qt/qt_common/map_widget.cpp

```diff
@@ -379,18 +379,14 @@ void MapWidget::initializeGL()
     }
   }
 #endif
-  auto fmt = context()->format();
-  if (m_apiOpenGLES3)
-  {
-    fmt.setProfile(QSurfaceFormat::CoreProfile);
-    fmt.setVersion(3, 2);
-  }
-  else
+
+  if (!m_apiOpenGLES3)
   {
+    auto fmt = context()->format();
     fmt.setProfile(QSurfaceFormat::CompatibilityProfile);
```

Comment:
Why is it needed to set the surface format yet again?
I mean it was needed on some legacy platform in the previous state of the code, but now the format is set in `main.cpp` no mather what the platform is.